### PR TITLE
feat(ui): Add sessions chart to project details

### PR DESCRIPTION
**Pull request by matejminar**
Adding new `Number of Sessions` chart to project details.

### CODE REVIEW

File: src/sentry/static/sentry/app/components/charts/releaseSeries.tsx [hunk 1]
```diff
@@ -17,7 +18,7 @@ import {escape} from 'app/utils';
 import {getFormattedDate, getUtcDateString} from 'app/utils/dates';
 import {formatVersion} from 'app/utils/formatters';
 import parseLinkHeader from 'app/utils/parseLinkHeader';
-import theme from 'app/utils/theme';
+import theme, {Theme} from 'app/utils/theme';
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/charts/releaseSeries.tsx:21]: now that the component is wrapped with `withTheme`, should we remove this import?

File: src/sentry/static/sentry/app/views/projectDetail/charts/sessionsRequest.tsx [hunk 2]
```diff
@@ -282,6 +302,32 @@ class SessionsRequest extends React.Component<Props, State> {
     };
   }
 
+  transformSessionCountData(
+    responseData: SessionApiResponse
+  ): {
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/projectDetail/charts/sessionsRequest.tsx:307]: do we need the output type?

**Review by priscilawebdev — APPROVED** [review]
I left 2 suggestions, but it looks good

File: src/sentry/static/sentry/app/views/projectDetail/charts/sessionsRequest.tsx (hunk 2 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/views/projectDetail/charts/sessionsRequest.tsx:307]: it helped me during development, but I can remove it now 👍

File: src/sentry/static/sentry/app/components/charts/releaseSeries.tsx (hunk 1 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/components/charts/releaseSeries.tsx:21]: good catch

**Pull request merged**